apis/guides/WixEmbedsAPI.md

```diff
@@ -0,0 +1,48 @@
+# WixEmbedsAPI
+The `wixEmbedsApi` is a property of the global `window` javascript object.
+
+Use this API in [self-hosted Embedded Script extensions](https://dev.wix.com/docs/build-apps/develop-your-app/frameworks/self-hosting/supported-extensions/site-extensions/add-an-embedded-script-extension-to-a-self-hosted-app) to interact with the site's page.
+
+## Usage
+You don't need to import `wixEmbedsApi` because it's a property of the global `window` javascript object. However, you can't immediately access it when your embedded script runs.
+
+Therefore, you should add an event listener that listens for the `wixEmbedsAPIReady` event.
+
+For example:
+```js
+if (window.wixEmbedsAPI) {
+  // Run your code that calls a wixEmbedsAPI method here.
+} else {
+  window.addEventListener(
+    "wixEmbedsAPIReady",
+    () => {
+      // Run your code that calls a wixEmbedsAPI method here.
+    },
+    false,
+  );
+}
+```
+
+You could also store a boolean variable that sets to `true` on `wixEmbedsAPIReady` and check the variable value before attempting to call a `wixEmbedsAPI` method.
+
+## Methods
+
+### getAccessTokenFunction()
+
+Returns an asynchronous function that returns a promise that resolves to an access token.
+
+Use this access token to call Wix [REST APIs](https://dev.wix.com/docs/rest) and [SDKs](https://dev.wix.com/docs/sdk) in an [embedded script](https://dev.wix.com/docs/build-apps/develop-your-app/frameworks/self-hosting/supported-extensions/site-extensions/add-an-embedded-script-extension-to-a-self-hosted-app) app extension. Learn more about [authorizing API calls in self-hosted embedded script extensions](ADD-LINK-TO-KB-ARTICLE).
+
```

Comment:
What kind of access token is this? Which identity does it use? Is that info included in the article you're linking out to?

Reply:
As this function is for use in self hosted apps, does that not mean it's site visitor auth?
In the [REST access token docs](https://dev.wix.com/docs/rest/app-management/oauth-2/introduction), it doesn't specify either the identity. Should we still include here?

Reply:
My guess is that it would be either site visitor or site member. Maybe we should mention that the access token allows you to authenticate as a site visitor or site member and link to [here](https://dev.wix.com/docs/build-apps/develop-your-app/access/about-identities)?